src/client/pages/ConsentsCommunicationPage.tsx

```diff
@@ -1,13 +1,6 @@
-import React, { useContext } from 'react';
-import { ClientState } from '@/shared/model/ClientState';
-import { ClientStateContext } from '@/client/components/ClientState';
+import React from 'react';
 import { ConsentsCommunication } from '@/client/pages/ConsentsCommunication';
 
 export const ConsentsCommunicationPage = () => {
-  const clientState = useContext<ClientState>(ClientStateContext);
-
-  const { pageData = {} } = clientState;
-  const { consents = [] } = pageData;
-
-  return <ConsentsCommunication consents={consents} />;
+  return <ConsentsCommunication />;
```

Comment:
If `pageData` is no longer used, likely there is code that can be cleaned up in `src/server/routes/consents.ts` which prepares this data.

Reply:
Similarly to my other comment, wondering if you've looked at this? Maybe not a blocker if you've got a follow-up PR coming though, as I don't think this will cause any noticable issues for users, it will just be firing an unneccessary request to fetch the consents.

Reply:
Thinking about it now, I actually think this was removed in error. Seems the consent information has been hardcoded into `src/client/pages/ConsentsCommunication.tsx, which means the copy won't change even if the response from IDAPI does.